projects/console-web-server/src/module/private/private-device.controller.ts

```diff
@@ -183,6 +183,22 @@ export class PrivateDeviceController {
     @Body() body: WriteDeviceRunTimeInfosRequestBody,
   ): Promise<void> {
     const { runtimeInfos } = body;
+    if (runtimeInfos.length === 0) {
+      return;
+    }
+    const lastInfo = runtimeInfos[runtimeInfos.length - 1];
+    if (0 < lastInfo.displays.length) {
+      const device = await this.deviceRepository.findOne({ where: { deviceId } });
+      if (device) {
+        const errorDisplay = lastInfo.displays.find((display) => display.error.length > 0);
+        if (!errorDisplay && device.displayError.length > 0) {
+          await this.deviceRepository.update({ deviceId }, { displayError: '' });
```

Comment:
If `displayError` type is `string | null`, update it to `null` would be more accurate

Reply:
@Ikalli I changed `displayError` to nullable! https://github.com/dogu-team/dogu/pull/410/commits/43b53c35920f2e26c11056c5d3224c39c5857c8a